README.md

```diff
@@ -1,31 +1,46 @@
-# OneConfig
+![OneConfig_banner](https://user-images.githubusercontent.com/62163840/193199250-6e28836d-cdb4-4967-9f6f-3f25e2dec19d.png)
 
-OneConfig is a next generation config library currently in development for all Minecraft mods.
 
-For a guide of how to use OneConfig please see our documentation at https://docs.polyfrost.cc
 
-![image](https://user-images.githubusercontent.com/67508414/172061081-7a32a139-d5c6-4f5a-907a-5af66deb799f.png)
-<br><br><br>
+# Introducing OneConfig
+Easy, stunning, expandable, open source. That sums it up quite well. We built OneConfig for the developer, to completly eliminate the hassle of making tools to configure their mods. Our API is elegant, and easy to learn. OneConfig was built from the ground up for this purpose. As for the user experience, it was our top priority. All of their mods will appear in one place, and they don't even need to install anything extra. The visual design is stunning, and is ready to grow whenever needed.
```

Comment:
'built oneconfig for the developer' and 'user experience was our top priority' clash a bit don't you think?

Reply:
maybe like user experience and dveleoper experience are our top priority